docs/source/changelog.md

```diff
@@ -5,7 +5,32 @@ its link will bring up a GitHub listing of changes. Use `git log` on the
 command line for details.
 
 
-## [Unreleased]
+## [Unreleased] 0.7
+
+### Added
+- Implement Services API [\#705](https://github.com/jupyterhub/jupyterhub/pull/705)
+- Add `/api/` and `/api/info` endpoints [\#675](https://github.com/jupyterhub/jupyterhub/pull/675)
+- Add documentation for JupyterLab, pySpark configuration, troubleshooting,
+  and more.
+- Add logging of error if adding users already in database.  [\#689](https://github.com/jupyterhub/jupyterhub/pull/689)
+
+### Changed
+- Always install with setuptools but not eggs (effectively require
+  `pip install .` [\#722](https://github.com/jupyterhub/jupyterhub/pull/722)
+- Updated formatting of changelog. [\#711](https://github.com/jupyterhub/jupyterhub/pull/711)
+- Use `/hub/login` instead of full URL in HubAuth.login_url and allow
+  absolute URLs if they are for our host (equivalent to absolute path)  [\#717](https://github.com/jupyterhub/jupyterhub/pull/717)
+- Install dependencies with pre [\#640](https://github.com/jupyterhub/jupyterhub/pull/640)
+
+### Fixed
+- Backward compatibility for ip, port in LocalProcessSpawner subclasses. [\#734](https://github.com/jupyterhub/jupyterhub/pull/748)
```

Comment:
This isn't a fix in 0.7, since it fixes something that was only introduced in master.